src/redux/configureStore.js

```diff
@@ -0,0 +1,8 @@
+import { configureStore } from '@reduxjs/toolkit';
```

Comment:
- [ ] Kindly note that this milestone does not require the use of `@reduxjs/toolkit`. Kindly follow the given specification. Kindly use the default `redux` package and **import** and make use of `createStore`, `combineReducers`, and _applyMiddleware_  from _redux_ in creating store.

Reply:
refactored code to use normal redux